.github/workflows/images-fork.yml

```diff
@@ -1,78 +0,0 @@
-name: Fork images
-env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-on:
-  workflow_dispatch:
-    inputs:
-      pr:
-        required: true
-        type: string
-        description: PR number
-
-jobs:
-  push-fork-image:
-    name: Deploy
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        suite:
-          - { name: bundler, ecosystem: bundler }
-          - { name: cargo, ecosystem: cargo }
-          - { name: composer, ecosystem: composer }
-          - { name: docker, ecosystem: docker }
-          - { name: elm, ecosystem: elm }
-          - { name: git_submodules, ecosystem: gitsubmodule }
-          - { name: github_actions, ecosystem: github-actions }
-          - { name: go_modules, ecosystem: gomod }
-          - { name: gradle, ecosystem: gradle }
-          - { name: hex, ecosystem: mix }
-          - { name: maven, ecosystem: maven }
-          - { name: npm_and_yarn, ecosystem: npm }
-          - { name: nuget, ecosystem: nuget }
-          - { name: pub, ecosystem: pub }
-          - { name: python, ecosystem: pip }
-          - { name: terraform, ecosystem: terraform }
-    permissions:
-      contents: read
-      packages: write
-    env:
-      TAG: ${{ github.sha }}
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Check if pull request is approved
-        # The PR must be approved. This is mostly to ensure you've typed the correct PR.
-        # Note: forks will have a blank review decision without approval. Not NEEDS_REVIEW.
```

Comment:
You probably want to copy this second sentence of the comment over to the new code? I don't see it there...

Reply:
I'm not sure I understand the comment. Since we are explicitly checking `DECISION == "APPROVED"`, does it matter what the default value for unreviewed PRs is? I guess that's why I didn't carry it over.

Reply:
That makes sense. I was thinking it was useful context for someone reviewing a failure and not understanding why
```
echo "Review decision is: $DECISION"
```

isn't spitting out NEEDS_REVIEW, I could see that being confusing during debugging.

I'm cool with what you've got as-is.